views/hud.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
your overindenting. please follow the style rules of https://github.com/nightscout/cgm-remote-monitor/blob/master/CONTRIBUTING.md#style-guide

I know see we don't apply them to most of the other static html files as well. If you can PR that within this PR please do so.